### PR TITLE
Update README.md

### LINKED ISSUE #37: Add View password feature to the Input tag
image

image

### PR DESCRIPTION
**Pull request by kdsinghcoder**
Previously Fixed #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,4 +64,6 @@ The user sees the following screen when during login
   - Inside [Handlebar](https://handlebarsjs.com/)
 - Database 
   - [MongoDB](https://docs.mongodb.com/) via [Mongoose](https://mongoosejs.com/)
+  
+
```
Comment (YashKumarVerma) [inline README.md:69]: please do not insert blank lines. Insert screenshots of the feature.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kdsinghcoder) [reply README.md:69]: ok @YashKumarVerma

**Comment by kdsinghcoder** [comment]
Changes are done and a new pull request is created. #47

**Comment by YashKumarVerma** [comment]
Closing as similar PR already merged.

**Pull request closed without merging**